explainers/signed-exchange-subresource-subtitution-explainer.md

```diff
@@ -0,0 +1,455 @@
+were prefetched in the referrer page. This is intended to avoid leaking the
+prefetching state to succeeding pages.
+- The UA check the header integrity value, so the distributor of the subresource
+signed exchange can’t inject arbitrary resources to the publisher’s page. This
+prevents distributors from sending tracking IDs to the publisher’s page.
+- The UA need to check both
+[the request URL](https://wicg.github.io/webpackage/loading.html#exchange-request-url)
+and the header integrity value of the signed exchange to avoid
+[Origin Laundering](https://hillbrad.github.io/sri-addressable-caching/sri-addressable-caching.html#origin_laundering)
+attack. The URL is not in signedHeaders, so the header integrity value can’t
+check the URL is correct.
+- The UA must fetch the signed exchange subresource
+(https://cdn.feed.example/cdn.publisher.example/lib.js.sxg) while processing the
+prefetch link element  (`<link rel="prefetch"
+href="https://cdn.feed.example/cdn.publisher.example/lib.js.sxg" as="script">`)
+even if there is the original subresource (http://cdn.publisher.example/lib.js)
+in the HTTPCache. Otherwise it leaks the state of  publisher’s site to the
+distributor of the signed exchange.
+- The UA must fetch the alternate signed exchange subresource (lib.js.sxg) even
+if there is the original subresource (lib.js) in the HTTPCache. Otherwise it
+leaks the state of publisher’s site to the distributor of the signed exchange.
+- To prevent tracking (user ID transfer), if the aggregator failed to prefetch a
+subresource that the main resource preloads, the UA must drop all of the
+subresource prefetches. If the aggregator prefetches a superset of the preloaded
+subresources, the UA must drop the ones that weren't preloaded.
+- The UA must ignore the not-yet-completed subresource signed exchange prefetch
+to prevent distributors from interfering the publisher’s page. (Eg.
+Intentionally block or delay the subresource loading.) So if the user clicks the
+link to the article while the UA is still prefetching the subresource signed
+exchange, the UA must fetch the subresource from the original URL after
+navigation.
+- The UA can use the prefetched signed exchange subresources only if the preload
+HTTP link headers for the subresources exist in the response headers. It is
+because the UA need to check the availability of all required signed exchanges
+before start loading subresources.
+
+## [Self-Review Questionnaire: Security and Privacy](https://www.w3.org/TR/security-privacy-questionnaire/)
+1. What information might this feature expose to Web sites or other parties, and
+for what purposes is that exposure necessary?
+   - The existence of the alternative signed exchange in HTTP Cache is exposed.
+1. Is this specification exposing the minimum amount of information necessary to
+power the feature?
+   - Yes. This proposal has limitations such as "all subresource SXG must be
```

Comment:
Say what the limitations accomplish here, not just that some phrases exist in other parts of this document.